src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -166,6 +151,40 @@ class SearchResultsPage extends React.Component {
             </div>
           </div>
 
+          <div className="nypl-full-width-wrapper">
+            <div className="nypl-row">
+              <div
+                className="nypl-column-three-quarters"
+                role="region"
```

Comment:
You can remove all the attributes in this `div` except for the `className` attribute. The others are not necessary.

Reply:
Removed the attributes as requested. Would be interesting to see how the toolkit changes work in conjunction with this project.